AirCasting/ForgotPassword/Views/ForgotPasswordView.swift

```diff
@@ -0,0 +1,56 @@
+// Created by Lunar on 15/07/2021.
+//
+
+import Foundation
+import SwiftUI
+import AirCastingStyling
+
+struct ForgotPasswordView<VM: ForgotPasswordViewModel>: View {
+    
+    @ObservedObject private var viewModel: VM
+    @Environment(\.presentationMode) var presentationMode
+    @State private var email = ""
+    
+    init(viewModel: VM) {
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 30) {
+            title
+            createTextfield(placeholder: viewModel.emailInputTitle, binding: $email)
+            description
+            sendButton
+        }.onChange(of: email, perform: { self.viewModel.emailChanged(to: $0) })
+        .padding()
+    }
+    
+    private var title: some View {
+        Text(viewModel.title)
+            .font(Font.moderate(size: 32, weight: .bold))
+            .foregroundColor(.accentColor)
+    }
+    
+    private var description: some View {
+        Text(viewModel.description)
+            .font(Font.muli(size: 16))
+            .foregroundColor(.aircastingGray)
+    }
+    
+    var sendButton: some View {
```

Comment:
I said it below but I'm adding this info here, too. Android app has two buttons "Ok" and "Cancel". Why did you decide to do it differently? ;)